ci/gather_diagnostics_data.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -eu -o pipefail
+
+echo "Services list: $1"
+
+IFS=',' read -ra SERVICES <<< "$1"
+
+if [ -z "${SERVICES}" ]; then
+  echo SERVICES is null or empty.
+  exit 1
+fi
+
+chmod +x ./python/create_service_diagnostics_bundle.sh
```

Comment:
Rather than chmod it here every time can you please change it in the repo?

Reply:
the x attribute, actually, is already there. This step just reproduce step from README.md.
anyway, I removed the line here.